feelpp/feel/feeltask/hardware.hpp

```diff
@@ -0,0 +1,260 @@
+namespace Feel
+{
+
+//================================================================================================================================
+// Get Information System CPU/GPU Hardware
+//================================================================================================================================
+
+namespace Hard {
```

Comment:
the namespace name is unclear, how about InformationSystem or SystemInformation ?

Reply:
but is the namespace necessary ?